ABloom/ABloom/Presentation/CoreViews/Home/ImageFileManager.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  ImageFileManager.swift
+//  ABloom
+//
+//  Created by Lee Jinhee on 10/29/23.
+//
+
+import PhotosUI
+
+class ImageFileManager {
```

Comment:
```suggestion
final class ImageFileManager {
```
상속될 여지가 없다면 **파이날 키워드** 가시죠!